test/e2e_node/podresources_test.go

```diff
@@ -568,6 +571,48 @@ func podresourcesGetAllocatableResourcesTests(ctx context.Context, cli kubeletpo
 	}
 }
 
+func podresourcesGetTests(ctx context.Context, f *framework.Framework, cli kubeletpodresourcesv1.PodResourcesListerClient) {
+	var err error
+	ginkgo.By("checking the output when no pods are present")
+	_, err = cli.Get(ctx, &kubeletpodresourcesv1.GetPodResourcesRequest{PodName: "test", PodNamespace: f.Namespace.Name})
+	framework.ExpectError(err, "pod not found")
+
+	tpd := newTestPodData()
+	ginkgo.By("checking the output when only pods which don't require resources are present")
+	expected := []podDesc{
+		{
+			podName: "pod-00",
+			cntName: "cnt-00",
+		},
+	}
+	tpd.createPodsForTest(ctx, f, expected)
+	resp, err := cli.Get(ctx, &kubeletpodresourcesv1.GetPodResourcesRequest{PodName: "pod-00", PodNamespace: f.Namespace.Name})
+	framework.ExpectNoError(err)
```

Comment:
this is now recognized as antipattern. Please add an annotation describing the failure. Example (feel free to change as you see fit): 
```
framework.ExpectNoError(err, "Get() call failed for pod %s/%s", f.Namespace.Name, "pod-00")
```
The idea is the failures should clearly point the reason and help the troubleshooting. Just using `framework.ExpectNoError(err)` is too terse and unhelpful.  Please make sure the `ExpectNoError` you add here are all annotated.

We/I will do later sweeping changes to this file to fix the other occurrences.

Reply:
(sorry for forgetting to add the labels earlier)

Reply:
I split it to 2 commits one add the describing  for ExpectNoError  to the old code and second commit with my changes.